go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	golang.org/x/sys v0.0.0-20200413165638-669c56c373c4 // indirect
 	golang.org/x/xerrors v0.0.0-20191204190536-9bdfabe68543 // indirect
 	k8s.io/api v0.18.0
+	k8s.io/apiextensions-apiserver v0.18.0
```

Comment:
please add this to the 1st `replace` section, so we use the same k8s version everywhere